packages/website/blog/2024-05-11-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -0,0 +1,310 @@
+  - If you have [`@typescript-eslint/no-empty-interface`](/rules/no-empty-interface) manually enabled, remove that, and instead either use a [recommended config](/users/configs) or manually enable [`@typescript-eslint/no-empty-object-type`](https://v8--typescript-eslint.netlify.app/rules/no-empty-object-type)
+
+### Tooling Breaking Changes
+
+- ⏳ [Enhancement: Error if configuration options aren't provided as expected](https://github.com/typescript-eslint/typescript-eslint/issues/6403)
+- [fix(typescript-estree): enable dot globs for project by default](https://github.com/typescript-eslint/typescript-eslint/pull/8818)
+  - This will cause any `parserOptions.project` globs to match dot (`.`) directories.
+    If you don't want to match them then use a more specific set of globs, or switch to `parserOptions.projectService`.
+- [feat(typescript-estree): remove slow deprecated and isolated programs](https://github.com/typescript-eslint/typescript-eslint/pull/8834)
+  - If you were still using `parserOptions.DEPRECATED__createDefaultProgram`, switch to `parserOptions.projectService` _(recommended)_ or `parserOptions.project`.
+- [feat(typescript-estree): rename automaticSingleRunInference to disallowAutomaticSingleRunInference](https://github.com/typescript-eslint/typescript-eslint/pull/8922)
+  - We've updated the default to be an opt-out - meaning you no longer need to enable it:
+    ```js
+    parserOptions: {
+      // Remove this line
+      automaticSingleRunInference: true,
+      // ...
+    }
+    ```
+- [chore: bump minimum versions for v8](https://github.com/typescript-eslint/typescript-eslint/pull/8973)
+  - ESLint support range was changed from `^8.56.0` to `^8.57.0`
+  - Node.js support range was changed from `^18.18.0 || >=20.0.0` to `^18.18.0 || ^20.9.0 || >=21.1.0`
+  - TypeScript support range was changed from `>=4.7.4 <5.5.0` to `>=4.8.4 <5.5.0`
+
+## Developer-Facing Changes
+
+typescript-eslint v8 comes with a suite of cleanups and improvements for developers as well.
+If you author any ESLint plugins or other tools that interact with TypeScript syntax, then we recommend you try out typescript-eslint v8 soon.
+It includes some breaking changes that you may need to accommodate for.
+
+:::tip
+If you're having trouble working with the changes, please let us know on [the typescript-eslint Discord](https://discord.gg/FSxKq8Tdyg)'s `#v8` channel!
+:::
+
+### AST Breaking Changes
+
+These PRs changed the AST shapes generated by typescript-eslint when parsing code.
+If you author any ESLint rules that refer to the syntax mentioned by them, these are relevant to you.
+
+- ⏳ [Enhancement: add strict parent types for nodes that have well-defined parents](https://github.com/typescript-eslint/typescript-eslint/issues/6225)
```

Comment:
```suggestion
- ⏳ [Enhancement: add strict parent types for nodes that have well-defined parents](https://github.com/typescript-eslint/typescript-eslint/issues/6225)
    - This will help you remove some unnecessary conditions - we suggest using `@typescript-eslint/no-unnecessary-condition` to help find the unnecessary checks!
```